File: PHP Excel Examples/CellsApiDemo/Row/insertRow.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "InsertRow.xlsx";
$folder = "Cells/Row";
$storage = null;
$sheetName = "Sheet1";
$rowIndex = 3;
$apiInstance->insertRow($name, $sheetName, $rowIndex, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/row/insertRow.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "InsertRow.xlsx"
sheetName = "Sheet1"
row_index = 3
storage = ""
folder = "/Cells/Row/"
api.insert_row(name, sheet_name=sheetName, row_index=row_index, folder=folder, storage=storage)